packages/debian/polymc/usr/share/applications/polymc.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=1.0.0
```

Comment:
`Version` key does not stand for the version of the application, but for the version of the desktop entry specification to which this file complies. It shouldn't have been changed from 1.0 to 1.0.0